modules/sovrnBidAdapter.js

```diff
@@ -158,11 +194,21 @@ export const spec = {
             dealId: sovrnBid.dealid || null,
             currency: 'USD',
             netRevenue: true,
-            mediaType: BANNER,
-            ad: decodeURIComponent(`${sovrnBid.adm}<img src="${sovrnBid.nurl}">`),
             ttl: sovrnBid.ext ? (sovrnBid.ext.ttl || 90) : 90,
             meta: { advertiserDomains: sovrnBid && sovrnBid.adomain ? sovrnBid.adomain : [] }
-          });
+          }
+
+          const videoRegex = new RegExp(/VAST\s+version/)
+
+          if (videoRegex.test(sovrnBid.adm)) {
+            bid.mediaType = VIDEO
+            bid.ad = decodeURIComponent(sovrnBid.adm)
```

Comment:
I am unsure if we want to set ad for video units, we probably want to decodeURI before the regex

Reply:
And we probably want to have uri decoded data in the VAST XML field

Reply:
Do we want to rely on a regex or should we reply with the type in our response @cpabst